protected/controllers/CsvController.php

```diff
@@ -237,4 +221,46 @@ public function createNewPatient($patient)
         return $errors;
     }
 
+    private function createNewTrialPatient($trial_patient)
+    {
+        $errors = array();
+        //trial
+        $trial = null;
+        if(isset($trial_patient['trial_name'])){
+            $trial = Trial::model()->findByAttributes(array('name' => $trial_patient['trial_name']));
+        }
+        if ($trial === null){
+            $errors[] = 'trial not found, please check the trial name';
```

Comment:
Shouldn't we bail out at this point? otherwise it will crash when the id is retrieved down below